app/helpers/filestack_rails/application_helper.rb

```diff
@@ -29,13 +29,17 @@ def filestack_transform
     end
 
     def filestack_image(url, options = {})
-      transform_object = options[:transform]
-      options.delete(:transform)
+      transform_object = options.delete(:transform)
+
+      image_tag(filestack_image_url(url, transform_object), options)
```

Comment:
@jeffdill2 Thank you for you PR, I will add it but I see that in your changes `transform_object` can be `{}` if options are not passed. So, `if transform_object` always return true.